code/game/objects/structures/xeno.dm

```diff
@@ -862,37 +860,70 @@ TUNNEL
 		if(EXPLODE_LIGHT)
 			take_damage(70)
 
+/obj/structure/xeno/acidwell/flamer_fire_act() //Removes a charge of acid, but fire is extinguished
+	acid_well_fire_interaction()
+
+/obj/structure/xeno/acidwell/fire_act() //Removes a charge of acid, but fire is extinguished
+	acid_well_fire_interaction()
```

Comment:
you should want them to be broken incase theres no charge though ?

Reply:
Takes fire damage now if there are no charges remaining.